action.yml

```diff
@@ -39,6 +39,15 @@ inputs:
   polling_timeout:
     required: false
     description: 'The duration (in milliseconds) after which the action stops polling for test results.'
+  fail_on_critical_errors:
+    required: false
+    description: 'Fail the CI job if no tests were triggered, or results could not be fetched from Datadog.'
+  fail_on_missing_tests:
+    required: false
+    description: 'Fail the CI job if at least one specified test with a public ID is missing in a run.'
```

Comment:
@alai97 for github actions, GitHub's UI shows a raw snippet of code based on this `action.yml` file.

On the right-hand side of the screenshot, you see that the descriptions are put as YAML comments in the generated example snippet:

![image](https://github.com/DataDog/synthetics-ci-github-action/assets/9317502/c652c82d-5454-46e3-8927-4a1bbeb8c74b)

IMO, we should not include a link there, otherwise it would be annoying to read, and scroll at the end of the line.